.gitignore

```diff
@@ -12,3 +12,6 @@
 # Visual studio temporaries, except a file used by QL4VS
 .vs/*
 !.vs/VSWorkspaceSettings.json
+
+# It's useful (though not required) to be able to unpack codeql in the ql checkout itself
+codeql/
```

Comment:
If it's only intended for the top level of the repo, I think it should be `/codeql/`.